### PR TITLE
Change the skeleton of cluster management

### PR DESCRIPTION
**Pull request by daniceexi**
I moved the [Hardware Architectures] to [Manage Clusters] since I
thought the name [Hardware Architectures] will make user confusing
that looks like this page is for hardware things instead of cluster 
management.

Another change was I moved the kvm part to be a sub-doc of specific
arch. This is because the kvm only can be done after the bare-metal
deployment. In the kvm part, I'd like only includes the [host install]
and [vm management]. The OS deployment should be same with bare-metal.

And since there's no much customer were using kvm, move it to low level
will make our doc more clear to follow.

### CODE REVIEW

**Comment by daniceexi** [comment]
The doc hierarchy after my change

image

**Comment by daniceexi** [comment]
I did not change x86_64 part, if we agree with the hierarchy change, I will change the part for x86_64 later.

**Comment by whowutwut** [comment]
@daniceexi , I did start off yesterday with IBM Power8 LE and OpenPower as the heading name, but then are we going to end up listing all the various hardware types?  What will the x86_64 heading change to?

File: docs/source/guides/admin-guides/manage_clusters/index.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Hardware Architectures 
-======================
+Manage Clusters
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/index.rst]: Does "Managing Clusters" make more sense here?

Should we add some text before the toc to describe something...

"Please choose one of the following supported hardware architectures for details in managing those cluster" .. or something like that

File: docs/source/guides/admin-guides/hardware_architectures/ppc64le/index.rst [hunk 2]
```diff
@@ -1,8 +0,0 @@
-ppc64le
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/hardware_architectures/ppc64le/index.rst]: I am okay for moving the kvm to be part of the power8le instructions.  I do think that we should still keep a subdirectory. What happens with other architectures?   Probably won't go backwards, but just using the older power hardwares as an example...
 
power7
power8be (tuletta) 
power8le 
power9le 

Right now it's under ppc64le/index.rst., that doesn't leave any room for creating additional architectures in the future

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/power8le_kvm/index.rst [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Manage kvm VM
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/power8le_kvm/index.rst]: Suggest "Manage KVM Virtual Machines" to be more clear.  KVM in caps

File: docs/source/guides/admin-guides/manage_clusters/ppc64le/power8le_kvm/index.rst [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+=================================
```
Comment (whowutwut) [inline docs/source/guides/admin-guides/manage_clusters/ppc64le/power8le_kvm/index.rst]: too many dashs here

**Comment by whowutwut** [comment]
@daniceexi , I found some issue with the bread-crumbs the way we are pointing to documents.  I think the last page that's rendered will have the bread-crumbs from that page.  If we look at the IBM power8 LE page... then click "next" which will go to "Hardware Discovery"

image

What we see is it's showing the x86_64 -> x86_64(kvm) page...

image


Then clicking "Back" will bring us to x86_64 (kvm)  ..  

I think we can't just link the document's but need to reference pages..    will work on this today.

**Pull request merged**